qc/workflow_exome.cwl

```diff
@@ -75,10 +75,40 @@ steps:
             metric_accumulation_level: picard_metric_accumulation_level
             bait_intervals: bait_intervals
             target_intervals: target_intervals
-            per_target_coverage: collect_hs_metrics_per_target_coverage
-            per_base_coverage: collect_hs_metrics_per_base_coverage
+            per_target_coverage:
+                default: false
+            per_base_coverage:
+                default: false
+        out:
+            [hs_metrics]
+    collect_per_base_metrics:
+        run: collect_hs_metrics.cwl
+        in:
+            cram: cram
+            reference: reference
+            metric_accumulation_level: picard_metric_accumulation_level
+            bait_intervals: bait_intervals
```

Comment:
I would suggest using the per_base_intervals for both the target_intervals and bait_intervals. Otherwise, the bait_intervals would be all over the genome/exome. The per-base intervals are likely individual sites that may or may not be covered by a bait. If we use the WES bait intervals, the Picard metrics for efficiency of design, etc. will also be skewed or not useful. The ultimate solution would be to intersect the baits with the per_base_interval input. However, I don't think it's necessary at this time.